contracts/DEX.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract DEX {
+    event CDTToSSVConverted(uint256 amount);
```

Comment:
we should think if an event is really necessary here.
if so, i would remove the token names here and make it more general.
also, we should add the user address and have both amounts (source and target).

Reply:
will need to discuss with Lior currently just for 2 our tokens

Reply:
Not sure we need an event